3rd_party/licenses/eigen-INFO.csv

```diff
@@ -1,2 +1,2 @@
 name,version,revision,url,license,copyright
-Eigen,3.3.7,323c052e17310984a40dda9426e0435bed7178fa,http://eigen.tuxfamily.org,MPL-2.0,
+Eigen,3.3.7,21ae2afd4edaa1b69782c67a54182d34efe43f9c,http://eigen.tuxfamily.org,MPL-2.0,
```

Comment:
The reason the commit hash changed is that Eigen switched from Mercurial to Git last December.  We are still using exactly the same code as before.